docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,847 @@
+{
```

Comment:
<p>The comma in the first sentence can be omitted</p><p> </p><p>For a tutorial, it would be important to explain the difference between a left MP and full MP. Then explain why a left MP is used to find discords instead of a full MP. Afterward, I would omit the parentheses and just write out "left nearest neighbor" and "left matrix profile" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>